gtsam/discrete/DecisionTree.h

```diff
@@ -299,6 +299,15 @@ namespace gtsam {
     /// Return the number of leaves in the tree.
     size_t nrLeaves() const;
 
+    /**
+     * @brief Return the number of total leaf assignments.
```

Comment:
Please give examples as to what this is, I think "number of assignments" needs to be explained.